### PR TITLE
fix: register KubeFed types

### PR DESCRIPTION
**Pull request by MatousJobanek**
Register KubeFed types with the scheme

### CODE REVIEW

File: pkg/apis/addtoscheme_kubefed_v1beta1.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package apis
+
+import "sigs.k8s.io/kubefed/pkg/apis/core/v1beta1"
+
+func init() {
+	// Register the KubeFed types with the Scheme so the components can map objects to GroupVersionKinds and back
+	AddToSchemes = append(AddToSchemes, v1beta1.SchemeBuilder.AddToScheme)
+}
```
Comment (nurali-techie) [inline pkg/apis/addtoscheme_kubefed_v1beta1.go]: This changes looks fine but I wanted to bring the change where rather adding one_new_file for each_new_scheme used .. we can just add new_scheme to `apis.go` file [link](https://github.com/codeready-toolchain/api/blob/master/pkg/apis/apis.go) directly.  Like we can just move this `init()` function to `apis.go` file.  Even we can delete `addtoscheme_toolchain_v1alpha1.go` file [link](https://github.com/codeready-toolchain/api/blob/master/pkg/apis/addtoscheme_toolchain_v1alpha1.go) and add its scheme to this `apis.go` file.

In short, one place (`apis.go`) to register all scheme rather one file for each scheme :-)

Comment (MatousJobanek) [reply pkg/apis/addtoscheme_kubefed_v1beta1.go]: I was thinking about it too. Actually, that was my original code (I mean for the case of KubeFedCluster), but then I decided to put it into a separated file. The reason was to keep the consistency with the `addtoscheme_toolchain_v1alpha1.go` - add the scheme in a separated file (one scheme per file). 
If you want to put everything into one file, then feel free to do it. The purpose of this PR was to register KubeFed scheme

Comment (MatousJobanek) [reply pkg/apis/addtoscheme_kubefed_v1beta1.go]: btw, the KubeFedCluster is deployed as part of our operators (in the same way as the other CRDs are), that's why I used the same approach as is used for toolchain types.
